### PR TITLE
Replace startBlock with startingBlock in genesis configurations

### PR DESCRIPTION
**Pull request by pirapira**
See https://github.com/ethereum/cpp-ethereum/pull/4511#discussion_r139375648

### CODE REVIEW

**Comment by pirapira** [comment]
Related leftover: detecting these spelling mistakes https://github.com/ethereum/cpp-ethereum/issues/4514

**Comment by pirapira** [comment]
@winsvega these are not cosmetic.  They might change the test results.

**Comment by winsvega** [comment]
hm, you right. Account.cpp:59 
how is it possible that precompiled tests went fine then?

**Comment by pirapira** [comment]
@winsvega we will see when the tests fail.

**Comment by winsvega** [comment]
I see. test networks not affected.

File: libethashseal/genesis/test/transitionnetTest.cpp [hunk 1]
```diff
@@ -56,10 +56,10 @@ R"E(
 		"0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
 		"0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
 		"0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } }
```
Comment (gumb0) [inline libethashseal/genesis/test/transitionnetTest.cpp]: comma missing in the end of the line here

File: libethashseal/genesis/test/mainNetworkNoProofTest.cpp [hunk 2]
```diff
@@ -55,10 +55,10 @@ R"E(
 		"0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
 		"0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
 		"0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
-		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startBlock" : "0x2dc6c0" } }
-		"0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "startBlock" : "0x2dc6c0", "linear": { "base": 500, "word": 0 } } },
-		"0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "startBlock" : "0x2dc6c0", "linear": { "base": 2000, "word": 0 } } },
-		"0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product", "startBlock" : "0x2dc6c0" } }
+		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startingBlock" : "0x2dc6c0" } }
```
Comment (gumb0) [inline libethashseal/genesis/test/mainNetworkNoProofTest.cpp]: comma missing in the end of the line here

File: libethashseal/genesis/test/constantinopleTransitionTest.cpp [hunk 3]
```diff
@@ -56,10 +56,10 @@ R"E(
 		"0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
 		"0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
 		"0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
-		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startBlock": "0x02" } }
-		"0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "startBlock": "0x02", "linear": { "base": 500, "word": 0 } } },
-		"0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "startBlock": "0x02", "linear": { "base": 2000, "word": 0 } } },
-		"0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product", "startBlock": "0x02" } }
+		"0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startingBlock": "0x02" } }
```
Comment (gumb0) [inline libethashseal/genesis/test/constantinopleTransitionTest.cpp]: comma missing in the end of the line here

**Comment by winsvega** [comment]
plese wait to see the test results

**Pull request merged**